madminer/limits/asymptotic_limits.py

```diff
@@ -84,7 +82,7 @@ def observed_limits(
         `x_observed` specifies the observed data as an array of observables, using the same observables and their order
         as used throughout the MadMiner workflow.
 
-        The p-values with frequentist hypothesis tests using the likelihood ratio as test statistic. The asymptotic
```

Comment:
["Frequentist"](https://en.wikipedia.org/wiki/Frequentist_inference) wasn't just a typo for "frequent" :) Could you revert this change, please?

Reply:
Yes. My bad